cluster/ubuntu/util.sh

```diff
@@ -508,6 +505,12 @@ function provision-masterandnode() {
       cp ~/kube/minion/* /opt/bin/
 
       service etcd start
+      service kube-apiserver start
```

Comment:
Don't bother bring up the k8s components one by one. They're triggered by `etcd`.